governator-core/src/main/java/com/netflix/governator/spi/README.md

```diff
@@ -0,0 +1 @@
+APIs for extending Governator
```

Comment:
Of course.  I wanted to make sure we're happy with the concept of an SPI and which classes go into the spi package before spending effort on the documentation.